eox_nelp/course_api/serializers.py

```diff
@@ -22,6 +23,7 @@ class NelpCourseDetailSerializer(CourseDetailSerializer):  # pylint: disable=abs
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        FuturexApiClient()
```

Comment:
Why this is added to nelp course detail serializer?

Reply:
my mistake